src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonSchemaCreateOptions.cs

```diff
@@ -23,6 +25,18 @@ public sealed class AIJsonSchemaCreateOptions : IEquatable<AIJsonSchemaCreateOpt
     /// </summary>
     public Func<AIJsonSchemaCreateContext, JsonNode, JsonNode>? TransformSchemaNode { get; init; }
 
+    /// <summary>
+    /// Gets a callback that is invoked for every parameter in the <see cref="MethodBase"/> provided to
+    /// <see cref="AIJsonUtilities.CreateFunctionJsonSchema"/> in order to determine whether it should
+    /// be included in the generated schema.
+    /// </summary>
+    /// <remarks>
+    /// By default, when <see cref="IncludeParameter"/> is <see langword="null"/>, all parameters other
+    /// than those of type <see cref="CancellationToken"/> are included in the generated schema.
```

Comment:
Nit: I thought the point of this delegate was to avoid hardcoding types getting filtered at the AIFunctionFactory layer. It can be reasonably argued that `CancellationToken` should always be filtered, but what if we want to exclude more parameter types in the future? (i.e. something like the context type we removed recently, which wouldn't be visible in this assembly)

Reply:
Right. My point is that this creates two sources of truth about what types do get filtered.